src/index.html

```diff
@@ -121,7 +121,10 @@ <h2 class="color-white">
     <script type="text/template" id="regionListTemplate">
         <div class="location--list" id="region-location-list">
             <p>
-                At consectetur, reprehenderit ad distinctio sapiente obcaecati facilis dolor possimus officia incidunt consequatur in sed dignissimos neque.
+              Coastal Resilience is a program led by The Nature Conservancy to examine nature’s role in
+              reducing coastal flood risk. The program consists of an approach, a web mapping tool, and
+              a network of practitioners around the world supporting hazard mitigation and climate
+              adaptation planning.
```

Comment:
I'd bet that this is a typo in the other version but the http://maps.coastalresilience.org/network/ appears not to have a period after `adaptation planning`. Probably fine to leave it here, but thought I'd note the slight mismatch.

Reply:
Good catch. Going to leave it.